src/main/java/fr/insalyon/citi/golo/compiler/JavaBytecodeGenerationGoloIrVisitor.java

```diff
@@ -451,14 +454,26 @@ public void visitThrowStatement(ThrowStatement throwStatement) {
     methodVisitor.visitInsn(ATHROW);
   }
 
-  private void visitInvocationArguments(AbstractInvocation invocation) {
-    for (ExpressionStatement statement : invocation.getArguments()) {
-      statement.accept(this);
+  private List<String> visitInvocationArguments(AbstractInvocation invocation) {
+    List<String> argumentNames = new ArrayList<>();
+    for (ExpressionStatement argument : invocation.getArguments()) {
+      if (invocation instanceof FunctionInvocation && ((FunctionInvocation) invocation).usesNamedArguments()) {
```

Comment:
I mean you put the named argument stuf on `ir.FunctionInvocation` not on `ir.AbstractInvocation`. So, I wonder if given e.g.
```golo
augment java.lang.String {

}
```
```golo
myobject: 
```